common/src/main/java/com/brandwatch/minibcr/common/domain/Mention.java

```diff
@@ -10,12 +10,12 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 
+@Data // Lombok: adds getters and setters
```

Comment:
Lombok is a useful tool, you can use it for this project. But for Brandwatch repositories we don't use it. Just giving you a heads up.